e2e/interchaintestv8/chainconfig/kurtosis.go

```diff
@@ -63,6 +64,7 @@ type kurtosisNetworkParams struct {
 	Participants        []kurtosisParticipant       `json:"participants"`
 	NetworkParams       kurtosisNetworkConfigParams `json:"network_params"`
 	WaitForFinalization bool                        `json:"wait_for_finalization"`
+	AdditionalServices  []string                    `json:"additional_services"`
```

Comment:
This is technically not related, but I did use to while testing to add an explorer to help me a bit, so figured we can just keep this around so it's easier to use.